scripts/check-signed-off-by.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+MAIN_BRANCH=master
+
+# ignores last commit because travis/gh creates a merge commit
+commits=$(git log --format=%h ${MAIN_BRANCH}..HEAD | tail -n +2)
+
+for sha in $commits; do
+  actual=$(git show -s --format=%B ${sha} | sed '/^$/d' | tail -n 1)
+  expected="Signed-off-by: $(git show -s --format="%an <%ae>" ${sha})"
```

Comment:
I wonder if we should also check the committer (%cn %ce). After github rebases, it'll just be github, but it would be nice to check this for when people apply other's patches, they should also be adding the Signed-off-by line.